### DISCUSSION TITLE
Labels are offset when running inference on clip

### DISCUSSION THREAD

**Discussion by timgjohnson** [talmolab/sleap#1450] [Help!]
Hi,

We noticed that when we run inference on a chunk of video, the labels seem to be offset by a frame. This is to say that the labels it will place on a given frame seem to line up really well with the next frame rather than the one they are actually placed on. This is especially exacerbated when they are moving.

2275
![image](https://github.com/talmolab/sleap/assets/137902068/9dd5d7d9-cdc9-4652-b24b-bbf3aaf0e32a)

2276
![image](https://github.com/talmolab/sleap/assets/137902068/7f7e6b03-3a20-45dc-b63e-e18a106026bb)

2277
![image](https://github.com/talmolab/sleap/assets/137902068/b48921dd-ea18-4bbf-a4bd-b893a5466624)

2278
![image](https://github.com/talmolab/sleap/assets/137902068/33f443ea-204b-4eb2-b97b-fa5ae6bcd9c0)

2279
![image](https://github.com/talmolab/sleap/assets/137902068/c8423d86-da50-43ea-9534-cd40fff1cb1d)

This offset doesn't seem present when running inferences on isolated frames. Would like to understand if that's something on our end/how to address it. It's awkward to scroll through when looking for track swaps, or trying to validate what our data analysis spits out.  

Thanks!

**Comment by talmo** [comment]
Hi @timgjohnson,

This seems like a classic video seekability issue. Check out [this help item](https://sleap.ai/help.html#i-get-strange-results-where-the-poses-appear-to-be-correct-but-shifted-relative-to-the-image) and let us know if the solution works for you.

Cheers,

Talmo

**Comment by timgjohnson** [comment]
Hi talmo,

Thanks for the advice, reencoding the video fixed the offset. If we were having that issue, does it affect our training frames? Like, would our labels be superimposed on a different frame from the one we see in the GUI when SLEAP tries to pull the labeled frames? Just because if that's the case, we probably want to switch out/relabel those videos.

Appreciate the help!
